rest/resource-server/src/test/java/org/eclipse/sw360/rest/resourceserver/restdocs/ObligationSpecTest.java

```diff
@@ -97,18 +99,31 @@ public void before() throws TException {
     public void should_document_get_obligations() throws Exception {
         String accessToken = TestHelper.getAccessToken(mockMvc, testUserId, testUserPassword);
         mockMvc.perform(get("/api/obligations")
-                .header("Authorization", "Bearer " + accessToken)
-                .accept(MediaTypes.HAL_JSON))
+                        .header("Authorization", "Bearer " + accessToken)
+                        .param("page", "0")
+                        .param("page_entries", "5")
+                        .accept(MediaTypes.HAL_JSON))
                 .andExpect(status().isOk())
                 .andDo(this.documentationHandler.document(
+                        requestParameters(
+                                parameterWithName("page").description("Page of licenses"),
+                                parameterWithName("page_entries").description("Amount of licenses per page")
+                        ),
                         links(
-                                linkWithRel("curies").description("Curies are used for online documentation")
+                                linkWithRel("curies").description("Curies are used for online documentation"),
+                                linkWithRel("first").description("Link to first page"),
+                                linkWithRel("last").description("Link to last page")
                         ),
                         responseFields(
                                 subsectionWithPath("_embedded.sw360:obligations[]title").description("The title of the obligation"),
                                 subsectionWithPath("_embedded.sw360:obligations[]obligationType").description("The obligationType of the obligation"),
                                 subsectionWithPath("_embedded.sw360:obligations").description("An array of <<resources-obligations, Obligations resources>>"),
-                                subsectionWithPath("_links").description("<<resources-index-links,Links>> to other resources")
+                                subsectionWithPath("_links").description("<<resources-index-links,Links>> to other resources"),
+                                fieldWithPath("page").description("Additional paging information"),
+                                fieldWithPath("page.size").description("Number of licenses per page"),
```

Comment:
change description of page_size to Number of obligations per page.

Reply:
Fixed